### PR TITLE
Drive improvements

### PR DESCRIPTION
**Pull request by albonec**
Deleted duplicate code, lmk if i missed anything unnecessary

### CODE REVIEW

**Review by arihant2math — CHANGES_REQUESTED** [review]
Don't delete the odometery stuff

**Comment by moogloof** [comment]
keep odometry

**Comment by moogloof** [comment]
ok good

**Review by moogloof — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Drive.java [hunk 1]
```diff
@@ -916,17 +793,6 @@ public void moveLeft(double distance, double motorSpeed) {
...
         //        sleep(100);
     }
-
-    /**
-     * Strafes the robot right by the specified distance with the default speed with odometry.
-     *
-     * @param distance The distance to strafe right by, in millimeters, use the mmPerInch constant if you want to use
-     *                 inches.
-     */
-    public void moveRightOdometry(double distance) {
```
Comment (arihant2math) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Drive.java:926]: This stuff if useful

**Comment by arihant2math** [comment]
Already merged

**Pull request closed without merging**